core/src/main/java/org/mskcc/oncotree/crosswalk/CrosswalkRepository.java

```diff
@@ -57,7 +64,7 @@ public MSKConcept getByOncotreeCode(String oncotreeCode)
     public MSKConcept queryCVS(String vocabularyId, String conceptId, String histologyCode, String siteCode)
             throws CrosswalkException {
         if (DISABLE_CVS_QUERYING) {
-            throw new CrosswalkServiceUnavailableException("CVS server queries have been disabled (crosswalk.disable_cvs_querying set to true)", new HttpServerErrorException(HttpStatus.SERVICE_UNAVAILABLE));
```

Comment:
This is a confusing to me -- we are still attempting a query despite querying being disabled. We are then relying on a separate boolean check to throw the exception. In this case is there a reason to retain the (`DISABLE_CVS_QUERYING`) check?

Reply:
I agree .. it is confusing. I guess I did not want to rip out all of the code for querying and caching crosswalk responses for the MSKConcept cache. So my thought here was to alter the meaning of crosswalk.disable_cvs_querying ... when false, an attempt is made to to query the (currently non-existent) external crosswalk server at data.mskcc.org ; when true we instead load the current production external references (NCI UMLS) from a static file.

It will be cleaner to remove the MSKConcept cache and fetch code from the codebase and the crosswalk.disable_cvs_querying configuration option. (but more work) Do you think that should be done?

Reply:
@sheridancbio is the Crosswalk system permanently down? if so, then I think it should be removed. But if it still a viable option then I think it should be kept and this static file approach be used as a separate method (triggered through a property)

Reply:
@averyniceday I'm pretty sure the system is down permanently .. but I also remember that there were no updates occurring for a long time even before it went offline. I'm guessing there is no value left there, so we should trim the code and drop the property - simply using the static file for now until a new solution for external references is developed.

Reply:
yes, then I think the other code should be trimmed out!